Sources/TuistKit/Services/Inspect/InspecImplicitImportsService.swift

```diff
@@ -31,7 +52,12 @@ enum InspectImplicitImportsServiceError: FatalError, Equatable {
     }
 }
 
-final class InspectImplicitImportsService {
+enum InspectType {
+    case redundant
+    case implicit
+}
+
+final class InspectImportsService {
```

Comment:
I'd consider keeping a service per command and use abstractions for the linting process itself.